### DISCUSSION TITLE
Resources for training

### DISCUSSION THREAD

**Discussion by ZJaume** [pemistahl/lingua-rs#286] [Show and tell]
In case that you are not aware, there is this wonderful repo including data for a lot of languages that Lingua does not support: https://github.com/laurieburchell/open-lid-dataset. All curated for LID purposes.

**Comment by pemistahl** [comment]
Thank you Jaume. I will keep this data set in mind even though there are probably better data sets for my library because open-lid uses the fasttext data format whereas my library uses a custom json format.
